1_fetch/src/fetch_wqp_data.R

```diff
@@ -0,0 +1,45 @@
+fetch_wqp_data <- function(wqp_inventory, characteristics, query_site_limit = 500, 
+                           min_results = 1, min_date = "", wqp_args = NULL){
+  
+  siteids <- wqp_inventory %>%
+    pull(MonitoringLocationIdentifier) %>%
+    unique()
```

Comment:
In the interest of having inputs represent the smallest amount of information needed to prevent unnecessary rebuilds, I'd suggest doing this step to get `siteids` from `wqp_inventory` outside of this function so that we are only passing in the unique siteids. Otherwise, the order that the site ids are listed or some other column value changes in `wqp_inventory` may kick off a full rebuild.

Reply:
OK, yeah that makes sense! Do you think it makes sense to create a separate target for the site id's?

Reply:
Yes, I think that would be good as a target!